conans/test/functional/toolchains/microsoft/test_msbuilddeps.py

```diff
@@ -115,9 +115,6 @@
   </ImportGroup>
   <ImportGroup Label="Shared">
   </ImportGroup>
-  <ImportGroup Label="PropertySheets">
```

Comment:
This test is testing exactly that different subprojects can depend on individual props, without needing to link all dependencies, please leave this test not modified.